sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Metrics/GetMetricSeriesDataOptions.cs

```diff
@@ -8,11 +8,17 @@
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
+    /// The set of options that can be specified when calling <see cref="MetricsAdvisorClient.GetMetricSeriesData"/>
+    /// or <see cref="MetricsAdvisorClient.GetMetricSeriesDataAsync"/> to configure the behavior of the request.
     /// </summary>
     public class GetMetricSeriesDataOptions
     {
         /// <summary>
+        /// Creates a new instance of the <see cref="GetMetricSeriesDataOptions"/> class.
         /// </summary>
+        /// <param name="seriesToFilter">Only time series with the specified series keys will be returned. These keys uniquely identify a time series within a metric. All possible dimensions of the associated <see cref="DataFeed"/> must be set.</param>
+        /// <param name="startTime">Filters the result. Only data points ingested after this point in time, in UTC, will be returned.</param>
+        /// <param name="endTime">Filters the result. Only data points ingested before this point in time, in UTC, will be returned.</param>
```

Comment:
nit - is the time inclusive or exclusive?

Reply:
The service doesn't make it clear. Some of them have the `the end timestamp of feedback timerange, when equals to startTime means only one timestamp` note in the swagger, as you pointed out. These make me think they are inclusive.

I'll update the wording accordingly:
> Only data points ingested ~after~ starting from this point...

> Only data points ingested ~before~ up to this point...